docs/dev/guides/backend-routes.md

```diff
@@ -111,6 +111,10 @@ Most of the time you probably want to add a menu entry for your back end module.
 Since the back end menu can be extended with an `EventListener` we can easily
 create one that listens for the menu event to be dispatched.
 
+{{% notice warning %}}
+The following functions works in Contao **>4.9** for the Version **4.9** use the BackendMenuListener.php from above.
```

Comment:
Don't you mean it only works in Contao 4.9 and previous versions need to use the example above?

Reply:
Yes!

Reply:
and above should probably be "below"  referring to the new block we added

Reply:
The original example only works for Versions up to 4.8 (before the menu refactoring), the example we added is for 4.9 and above.